torch/csrc/jit/passes/quantization.cpp

```diff
@@ -697,18 +728,31 @@ void InsertQuantDeQuantHelper::removeObservers(
     }
     nodes_to_destroy_.at(g).clear();
   }
+
+  // If we have seen this graph before, we'll replay the observer
+  // slots removal using the slot index
```

Comment:
I couldn't understand this comment :) I understood what the code does after reading it, but the comment itself didn't help.

Reply:
Which part is confusing?
I'll write more detailed comment.